index.ts

```diff
@@ -15,8 +19,12 @@ export default function (items: any[], callback: (item: any, index?: number) =>
     approveFunctions.forEach((key) => {
       const appliedApproveFunctions = callback(item, i);
       const approveFunction = appliedApproveFunctions[key];
+      const isFunction = typeof approveFunction === 'function';
```

Comment:
You did not use this variable down below. The if condition could be rewritten with a simple ternary with this variable. :)

Reply:
Oops! A remnant of a previous iteration, cheers :grin: